Workshop/test_framework.rb

```diff
@@ -0,0 +1,125 @@
+require "test/unit"
+require_relative "framework"
+
+class Test_Website < Test::Unit::TestCase
+
+	def test_make_object
+		framework = Framework.new()
+        assert framework.instance_of? Framework
+    end
+
+	def test_get_title
+		framework = Framework.new()
+		framework.open_browser("file:///app/index.html")
+		assert_equal("Document", framework.get_title())
+		framework.close_browser()
+	end
+
+	def test_get_element_by_id()
+		framework = Framework.new()
+		framework.open_browser("file:///app/index.html")
+		assert_equal("", framework.get_text(framework.find_element_by_id("name")))
+		framework.close_browser()
+	end
+
+	def test_negative_get_element_by_id()
+		framework = Framework.new()
+		framework.open_browser("file:///app/index.html")
+		assert_equal("Error", framework.get_text(framework.find_element_by_id("names")))
+		framework.close_browser()
+	end
+
+	def test_get_element_by_class()
+		framework = Framework.new()
+		framework.open_browser("file:///app/index.html")
+		assert_equal("Contact Us Today!", framework.get_text(framework.find_elements_by_class("legend")[0]))
+		framework.close_browser()
+	end
+
+	def test_negative_get_element_by_class()
+		framework = Framework.new()
+		framework.open_browser("file:///app/index.html")
+		assert_equal("Error", framework.get_text(framework.find_elements_by_class("legends")[0]))
+		framework.close_browser()
+	end
+
+	def test_get_element_by_tag_name()
+		framework = Framework.new()
+		framework.open_browser("file:///app/index.html")
+		assert_equal("Contact Us Today!", framework.get_text(framework.find_elements_by_tag_name("legend")[0]))
```

Comment:
Each unit test should test only *one* function. This unit test uses the `get_text` and `find_elements` method.

Reply:
You can use the driver methods to get the text

Reply:
Change other unit tests similarly